config/routes.rb

```diff
@@ -1,6 +1,6 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ("/")
-  # root "articles#index"
+  resources :flights, only: [:index] do
+    resources :flight_passengers, only: [:destroy]
+  end
```

Comment:
Although you are making use of advanced routing here, what you have here is incorrect. `FlightPassengers` is not a child of `Flights`. The path is incorrect.